### DISCUSSION TITLE
Tailwindcss performance optimization about using variable in className

### DISCUSSION THREAD

**Discussion by Dann1y** [tailwindlabs/tailwindcss#9612] [Help]
I wonder if I use variable in className, is there have any performance difference when compile or any other situation.

For example,
//responsiveWidth is variable
`<div style={{ width: responsiveWidth  + '%' }} >{some codes..}</div>`

or 

`<div className=`w-[${responsiveWidth}%]`>{some codes...}</div>`

**Comment by wongjn** [comment]
[As per the documentation](https://tailwindcss.com/docs/content-configuration#dynamic-class-names), you cannot have dynamic class names like your second option.
